app/src/main/java/com/saraswitty/wittyapp/AuthenticatedActivity.kt

```diff
@@ -97,14 +98,36 @@ class AuthenticatedActivity : AppCompatActivity() {
                 PendingIntent.FLAG_UPDATE_CURRENT
         )
 
+        // Snooze Intent for the snooze button in the notification
+        val snoozeIntent = Intent(this, SnoozeActivity::class.java)
+        val snoozePendingIntent: PendingIntent =
+                PendingIntent.getActivity(this, 1, snoozeIntent, PendingIntent.FLAG_UPDATE_CURRENT)
+
+        // Event Intent for the sign me up button in the notification
+        val checkEventPageIntent = Intent(this, NewUpcomingEvent::class.java)
+        val checkEventPagePendingIntent: PendingIntent =
+                PendingIntent.getActivity(this, 1, checkEventPageIntent, PendingIntent.FLAG_UPDATE_CURRENT)
+
         // Use NotificationCompat instead of Notification for backward compatibility
         val notification = NotificationCompat.Builder(this@AuthenticatedActivity,
                 channelID)
-                .setContentTitle("Example Notification")
-                .setContentText("This is an  example notification.")
-                .setSmallIcon(android.R.drawable.ic_dialog_info)
+                .setContentTitle("New Event")
+                .setContentText("This is an example event.")
+                .setLargeIcon(BitmapFactory.decodeResource(resources, R.drawable.sneh_large_icon))
+                .setSmallIcon(R.drawable.sneh_small_icon)
                 .setChannelId(channelID)
                 .setContentIntent(pendingIntent)
+                .addAction(R.drawable.sneh_small_icon, "Sign Me Up",
```

Comment:
Can we skip the small icon on the action buttons? Just the text 'Sign me up' should be good enough.